### ISSUE TITLE
Question 4 - solved

### ISSUE THREAD

**Issue by thao-nguyen-10** [thao-nguyen-10/machine-learning-course-hcmut#4]
Implement batch gradient descent to find the optimal weights for predicting CreditScore using Age as input. Starting with initial parameters θ₀=500, θ₁=5, compute the cost function and one iteration of gradient descent updates using learning rate α=0.01. Interpret the direction of the parameter updates.

**Comment by thao-nguyen-10** [comment]
![Image](https://github.com/user-attachments/assets/896b7d7c-cf26-470b-9d69-4cb6c1d44118)

**Comment by thao-nguyen-10** [comment]
![Image](https://github.com/user-attachments/assets/b6718917-13f6-488d-b22c-0e7ca11e5492)
